README_VERITABLE.md

```diff
@@ -34,12 +34,28 @@ The Aries Framework JavaScript REST API is the most convenient way for self-sove
 - 💻 **CLI** that makes it super easy to start an instance of the REST API.
 - 🌐 **Interoperable** with all major Aries implementations.
 
+## Description
+
+The project aims to enable supply chains to share insights and data across multiple data platforms, enhancing the efficiency of the supply chain network. Specifically it facilitates the onboarding of new actors into a network, facilitationg connections between those and allows for interaction between the actors. Interactions include issuance of credentials, acceptance of issued credentials, requesting proofs of credentials from other actors and those being able to verify those requests.
+
 ## Quick start
 
 The REST API provides an OpenAPI schema that can easily be viewed using the SwaggerUI that is provided with the server. The docs can be viewed on the `/docs` endpoint (e.g. http://localhost:3000/docs).
 
+Bellow you will find commands for starting up the containers in docker (see the [Using Docker](#using-docker-easiest) section) or via cli (see [Via Cli](#via-cli)).
+
 > The OpenAPI spec is generated from the model classes used by Aries Framework JavaScript. Due to limitations in the inspection of these classes, the generated schema does not always exactly match the expected format. Keep this in mind when using this package. If you encounter any issues, feel free to open an issue.
 
+### Rest Client Args
+
+The CLI args are defined under `src > cli.ts ` They are used to start up a service.
+| Argument added | Required | Default | Description |
+| :------------- | :------: | :------ | :------- |
+| ipfs-origin | Y | http://ipfs0:5001 | IPFS endpoint |
+|opa-origin |Y |http://opa:8181 |OPA endpoint |
```

Comment:
I've realised this is missing in `cli.ts` could you please add it as an option?
```
  .option('opa-origin', {
    string: true,
    default: 'http://localhost:8181'
  })
```

Reply:
Ty @jonmattgray, it's been added, so should I change the opa-origin default to `http://localhost:8181` ? and does that mean that it is not required? 
Equally should I change the ipfs-origin not to have a default?

Reply:
talked in standup